<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查找内容</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TextOpen.qrc">
    <normaloff>:/images/logo.ico</normaloff>:/images/logo.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>查找内容:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>区分大小写</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>全字匹配</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>查找上一个</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>查找下一个</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>正则表达式</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="TextOpen.qrc"/>
 </resources>
 <connections/>
</ui>
